test/e2e/autoscaling/dns_autoscaling.go

```diff
@@ -119,9 +116,7 @@ var _ = SIGDescribe("DNS horizontal autoscaling", func() {
 
 	// This test is separated because it is slow and need to run serially.
 	// Will take around 5 minutes to run on a 4 nodes cluster.
-	// TODO(upodroid) This test will be removed in 1.33 when kubeup is removed
-	// TODO: make it cloud provider agnostic or move it to cloud-provider-gcp repository
-	f.It(f.WithSerial(), f.WithSlow(), f.WithLabel("KubeUp"), f.WithLabel("sig-cloud-provider-gcp"), "kube-dns-autoscaler should scale kube-dns pods when cluster size changed", func(ctx context.Context) {
+	f.It(f.WithSerial(), f.WithSlow(), "kube-dns-autoscaler should scale kube-dns pods when cluster size changed", func(ctx context.Context) {
```

Comment:
this will need something like a Feature label because clusters don't necessarily have a kube-dns-autoscaler so this test shouldn't run by default. @aojea thoughts?

Reply:
yeah, you can declare a new feature https://github.com/kubernetes/kubernetes/blob/9e42e08547375ca217c2fccad46337985d97cd85/test/e2e/feature/feature.go#L43-L74. `kube-dns-autoscaler` and tag the test